Formula/nginx.rb

```diff
@@ -18,6 +18,14 @@ class Nginx < Formula
   depends_on "pcre"
 
   def install
+    # keep clean copy of source for compiling dynamic modules e.g. passenger
+    (share/"src").mkpath
+    mkdir "src_rebuild" do
+      system "tar", "-xf", cached_download, "--strip-components", "1", "--exclude='conf contrib LICENSE README html man CHANGES.ru CHANGES'"
+      system "tar", "-cJf", "src.txz", "--options", "compression-level=9", "--exclude", "src.txz", "."
```

Comment:
Specify the full output path instead of doing the `.install` below. You can also avoid the `--exclude` that way. Let's also go for `.tar.xz` as it seems more widely used.

Reply:
done